opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -279,33 +279,39 @@ namespace Opm
 
             // compute volume ratio between connection at standard conditions
             Value volumeRatio = bhp * 0.0; // initialize it with the correct type
-;
+
             if (FluidSystem::enableVaporizedWater() && FluidSystem::enableDissolvedGasInWater()) {
```

Comment:
This is not in this PR, so I approved the PR. 

A discussion here, should here be 
```c++
if (FluidSystem::enableVaporizedWater() || FluidSystem::enableDissolvedGasInWater()) {
```
??